test/nullable.jl

```diff
@@ -387,3 +387,7 @@ end
 
 # issue #11675
 @test repr(Nullable()) == "Nullable{Union{}}()"
+
```

Comment:
Could you add a comment with the PR/issue number? That's really useful when you want to understand what is being tested exactly.

Otherwise, LGTM.